cmd/hz/util/env.go

```diff
@@ -103,6 +103,10 @@ func SearchGoMod(cwd string, recurse bool) (moduleName, path string, found bool)
 			break
 		}
 		cwd = filepath.Dir(cwd)
+		// "C:\" will return "C:\" under windows
+		if cwd == filepath.Dir(cwd) {
```

Comment:
What is the purpose of this check? Maybe adding more description in comment?

Reply:
Not determining the root directory under windows， it can lead to a dead loop